### ISSUE TITLE
Hello-world-svelte register allComponents does not seems to do anything

### ISSUE THREAD

**Issue by Askhento** [microsoft/vscode-webview-ui-toolkit-samples#137]
### Describe the bug

Using allComponents does not work

### To reproduce

In App.svelte
```svelte
<script>
//   provideVSCodeDesignSystem().register(vsCodeButton());
//   provideVSCodeDesignSystem().register(vsCodeOption());
  provideVSCodeDesignSystem().register(allComponents.register());
</script>

<main>
  <h1>Hello world!</h1>
  <vscode-button on:click={handleHowdyClick}>Howdy!</vscode-button>
  <vscode-option selected>OPTIONNNN</vscode-option>
</main>
```

### Screenshots
Result is : 
<img width="207" alt="image" src="https://user-images.githubusercontent.com/34323808/222421424-13e28f73-a0be-4bb3-abce-7375fd99dcdb.png">

If added elements one by one : 
<img width="221" alt="image" src="https://user-images.githubusercontent.com/34323808/222422074-42a38446-474c-4a74-88f4-335b20884aba.png">

### Desktop (please complete the following information):

-   OS Version:  macOS 10.15.7

**Comment by hawkticehurst** [comment]
Hey @Askhento!

Whoops, I think this is a typo in all the extension samples/docs 😅 Thank you for the catch! 

The correct method call is:

```js
provideVSCodeDesignSystem().register(allComponents);
``` 

I'll be out of office until the end of the month but will update all the sample extensions/docs once I'm back

**Comment by KaruroChori** [comment]
https://github.com/microsoft/vscode-webview-ui-toolkit-samples/pull/132

**Comment by hawkticehurst** [comment]
This issue was resolved in #132
